main.go

```diff
@@ -55,12 +55,24 @@ terraform {
 	backend "s3" {}
 }
 
-variable "name" {}
-variable "organization" {}
-variable "terraform_version" {}
+variable "name" {
+	type = string
+}
+variable "organization" {
+	type = string
+}
+variable "terraform_version" {
+	type = string
+}
+variable "workspaces" {
```

Comment:
Worth considering whether it's even necessary to accept name + workspaces or just compute the workspace names in Go. The advantage of the latter is that you can check whether the workspaces already exist for the sake of imports.

Reply:
true yeah, for importing it would be nice to compute once and pass in the result.

I saw the update for the [Jira ticket](https://takescoop.atlassian.net/browse/PLATFORM-2286), how would we account for other resources in that import scenario? Not that the other resources are in danger of being deleted, you'd just get a conflict which would need to be addressed somehow

Reply:
Try to import everything. By trying to discover importable resources by name, relation, etc., and constructing the `import` commands.

Reply:
okay lets give it a shot. i'll reconfigure workspace and names